internal/resource/config/idp/spconnection/idp_sp_connection_resource.go

```diff
@@ -668,9 +586,8 @@ func (r *idpSpConnectionResource) Schema(ctx context.Context, req resource.Schem
 						},
 					},
 					"character_case": schema.StringAttribute{
-						Optional: true,
-						Computed: true,
-						//Default:     stringdefault.StaticString("NONE"),
```

Comment:
I think I wanted to leave these defaults in with the comments to that we can remove the comments when Terraform fixes things